packages/pretty-format/src/__tests__/prettyFormat.test.ts

```diff
@@ -41,6 +41,26 @@ describe('prettyFormat()', () => {
     expect(prettyFormat(val)).toEqual('Array [\n  1,\n  2,\n  3,\n]');
   });
 
+  it('prints a sparse array with only holes', () => {
+    // eslint-disable-next-line no-sparse-arrays
+    const val = [, , ,];
+    expect(prettyFormat(val)).toEqual('Array [\n  ,\n  ,\n  ,\n]');
+  });
+
+  it('prints a sparse array with items', () => {
```

Comment:
could you add a test where the value is in the middle surrounded by holes as well?

Reply:
Will do